src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -3082,7 +3082,18 @@ private void UpdateDatabasesWithNewServerWideBackupConfiguration(TransactionOper
                     {
                         foreach (BlittableJsonReaderObject backup in backups)
                         {
-                            if (IsServerWideBackupWithTaskName(backup, periodicBackupConfiguration.Name))
```

Comment:
1) Shouldn't this method ensure that the configuration name starts with `ServerWideBackupConfiguration.NamePrefix` ?
2) You removed the usage of this method here but there is another one in `DeleteServerWideBackupConfigurationFromAllDatabases` is its usage correct there?

Reply:
I didn't remove the behavior of the method just the usage.
I just wanted to use the backup name twice for the remove check and for the edit check
and while the code in `IsServerWideBackupWithTaskName` is very small so there is a small benefit of reuse it.

By the way, I thought to get the name as `LazyStringValue` but I didn't find an overload of `LazyStringValue.Equal` with `StringComparison` and I don't know if it so relevant here